src/dev/notice/generate_notice_text.js

```diff
@@ -0,0 +1,57 @@
+import vfs from 'vinyl-fs';
+
+import { REPO_ROOT } from '../constants';
+
+const NOTICE_COMMENT_RE = /\/\*[\s\n\*]*@notice([\w\W]+?)\*\//g;
+const NEWLINE_RE = /\r\n|[\n\r\u2028\u2029]/g;
+
+export async function generateNoticeText() {
+  const files = vfs.src([
+    '**/*.{js,less,css}',
+  ], {
+    cwd: REPO_ROOT,
+    nodir: true,
+    ignore: [
+      '{node_modules,build,target,dist}/**',
```

Comment:
Yeah, I definitely struggled with this, since the task is run in CI I don't really think we should be logging every file but I'll add logging for files that are detected to have notices.

Reply:
> I'll add logging for files that are detected to have notices.

Yep, that is what I meant :)